addons/pos_gift_card/static/src/js/models.js

```diff
@@ -36,9 +36,6 @@ odoo.define("pos_gift_card.gift_card", function (require) {
         orderline.generated_gift_card_ids = [options.generated_gift_card_ids];
       }
       if (options && options.gift_card_id) {
-        if (orderline.order.orderlines.find((line) => line.gift_card_id === options.gift_card_id)) {
```

Comment:
why this snippet was deleted?

Reply:
As I explained in the commit and description of the PR, we are creating more than one line with the same gift card (just splitting the amount by taxes) and this validation didn't allow us to do that.

Also as a note in v16, this validation doesn't exist